alma/lib/Model/CartData.php

```diff
@@ -166,7 +166,7 @@ public function getCartItems($cart)
                 'unit_price' => $this->priceHelper->convertPriceToCents($unitPrice),
                 'line_price' => $this->priceHelper->convertPriceToCents($linePrice),
                 'is_gift' => $isGift,
-                'categories' => [$productRow['category']],
+                'categories' => $this->productHelper->getCategoryName($productRow),
```

Comment:
I think you can use "categories" in the plural in getGategoryName ( getCategoriesNames)